advocacy_docs/pg_extensions/query_advisor/index.mdx

```diff
@@ -34,13 +34,13 @@ See [Using EDB Query Advisor](using/#query_advisor_index_recommendationsmin_filt
 
 ## Statistics Advisor
 
-Statistics play a crucial role in determining the quality of query execution plans chosen by the database query optimizer. Sometimes there are dependencies between columns, and unless we create combined statistics using extended statistics, the optimizer assumes these columns are independent. This assumption often leads to incorrect cardinality estimates, which in turn, results in a poor execution plan, negatively impacting query performance.
+Statistics play a crucial role in determining the quality of query execution plans chosen by the database query optimizer. Sometimes there are dependencies between columns, and unless combined statistics are created using extended statistics, the optimizer assumes these columns are independent. This assumption often leads to incorrect cardinality estimates, which in turn, results in a poor execution plan, negatively impacting query performance.
 
-To address this, we monitor queries where multiple-column filters are applied and check for any selectivity estimation errors. If such errors are detected, we treat those multi-column filters as potential candidates for extended statistics.
+To address this, Statistics Advisor monitors queries where multiple-column filters are applied and checks for any selectivity estimation errors. If such errors are detected,those multi-column filters are treated as potential candidates for extended statistics.
 
-Whenever extended statistics recommendations are generated, we process these candidates by exploring various combinations of columns, especially when more than two columns are involved in the filters. Since estimation errors alone don’t definitively indicate which columns are dependent, we try all possible combinations. Currently, to limit the candidate pool, we focus on statistics for no more than two columns at a time.
+Whenever extended statistics recommendations are generated, those candidates are processed by exploring various combinations of columns, especially when more than two columns are involved in the filters. Since estimation errors alone don’t definitively indicate which columns are dependent, Statistic Advisor tries all possible combinations. Currently, to limit the candidate pool, the focus on statistics is for no more than two columns at a time.
```

Comment:
```suggestion
Whenever extended statistics recommendations are generated, Statistics Advisor processes those candidates by exploring various combinations of columns, especially when more than two columns are involved in the filters. Since estimation errors alone don’t definitively indicate which columns are dependent, Statistic Advisor tries all possible combinations. Currently, to limit the candidate pool, the focus on statistics is for no more than two columns at a time.
```